packages/js/src/plugins/auctionHouseModule/createAuctionHouse.ts

```diff
@@ -149,5 +156,41 @@ export const createAuctionHouseBuilder = (
       ),
       signers: [payer],
       key: params.instructionKey ?? 'createAuctionHouse',
+    })
+    .when(Boolean(params.auctioneerAuthority?.publicKey), (builder) => {
+      const auctioneerAuthority = params.auctioneerAuthority
+        ?.publicKey as PublicKey;
+      const ahAuctioneerPda = findAuctioneerPda(
+        auctionHouse,
+        auctioneerAuthority
+      );
+
+      const scopes = params.auctioneerScopes ?? [
+        AuthorityScope.Deposit,
+        AuthorityScope.Buy,
+        AuthorityScope.PublicBuy,
+        AuthorityScope.ExecuteSale,
+        AuthorityScope.Sell,
+        AuthorityScope.Cancel,
+        AuthorityScope.Withdraw,
+      ];
+
+      builder.add({
+        instruction: createDelegateAuctioneerInstruction(
+          {
+            auctionHouse,
+            authority,
+            auctioneerAuthority,
+            ahAuctioneerPda,
+          },
+          {
+            scopes,
+          }
+        ),
+        signers: [payer],
+        key: params.instructionKey ?? 'delegateAuctioneer',
```

Comment:
This should be a different parameter. Each instruction should be recognisable by a unique key that can be provided as input.

```suggestion
        key: params.delegateAuctioneerInstructionKey ?? 'delegateAuctioneer',
```